smsdk/ma_session.py

```diff
@@ -248,15 +248,9 @@ def _complete_async_task(
                     if state == "FAILURE" or state == "REVOKED":
                         raise ValueError("Error - {}".format(response.text))
                 except:
```

Comment:
```suggestion
                except Exception as e:
                  raise e
```
This doesn't feel like returning the response is fixing the issue the ticket suggests that we should be raising the actually error.  Maybe print the error as log line as well.

Reply:
Done